### DISCUSSION TITLE
Welcome to D3FEND Ontology Discussions!

### DISCUSSION THREAD

**Discussion by netfl0** [d3fend/d3fend-ontology#129] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dsdr0** [comment]


**Reply by netfl0** [reply]
@dsdr0, please create this as a separate/new discussion.
